### PR TITLE
basic transferLocked erc20

### PR DESCRIPTION
**Pull request by ZrowGz**
Adds functionality allowing transfer of locked ERC20 assets without withdrawing. 

Utilizes the same kek_id, but this could be changed to rehash this value if needed.

### CODE REVIEW

**Comment by FortisFortuna** [comment]
There could be unforseen problems by reusing kek_ids. I would prefer to have each stake have a unique one

**Comment by FortisFortuna** [comment]
Looks good assuming Convex side is correct and doesn't need anything else here.

**Comment by ZrowGz** [comment]
Testing completed & updates made. 

Testing was performed in a separate repository to be able to utilize the modified version of the convex vault simultaneously. Testing found here: https://github.com/ZrowGz/frax-transfers.git

File: src/hardhat/contracts/Staking/FraxUnifiedFarm_ERC20.sol [hunk 1]
```diff
@@ -25,7 +25,7 @@ import "./FraxUnifiedFarmTemplate.sol";
 // import "../Misc_AMOs/curve/I2poolToken.sol";
 
 // Fraxswap
-import '../Fraxswap/core/interfaces/IFraxswapPair.sol';
```
Comment (regalwright) [inline src/hardhat/contracts/Staking/FraxUnifiedFarm_ERC20.sol]: import '../Fraxswap/core/interfaces/IFraxswapPair.sol';

**Pull request merged**